frontend/src/store/app/types.ts

```diff
@@ -38,21 +34,14 @@ interface SetLocaleAction {
   locale: string;
 }
 
-interface SetClock {
-  type: typeof CLOCK_UPDATED;
-}
-
 interface SetRedBorder {
   type: typeof RED_BORDER;
   status: boolean;
 }
 
-interface UpdateBackendHeartbeat {
-  type: typeof BACKEND_HEARTBEAT;
-}
-
 interface BackendConnectionDown {
   type: typeof BACKEND_CONNECTION_DOWN;
+  update: Record<string, unknown>;
```

Comment:
What is update supposed to contain?

Reply:
This is the part where we pass LogEventType. code, and time! looking at your other comment we should be doing this in the reducer then this can be removed.

Reply:
deleted it therefore resolving